src/config/index.ts

```diff
@@ -39,6 +39,21 @@ export const getNetworkLabel = (id: ETHEREUM_NETWORK): string => {
   return cfg ? cfg.network.label : ''
 }
 
+// Returns true if the address points to a GoldToken proxy
+// on a support Celo network.
+export const isGoldTokenAddress = (address: string): boolean => {
+  switch (getNetworkId()) {
+    case ETHEREUM_NETWORK.ALFAJORES:
+      return address.toLowerCase() === '0xF194afDf50B03e69Bd7D057c1Aa9e10c9954E4C9'.toLowerCase()
```

Comment:
These won't match up. The string on the right has uppercase characters.

Reply:
there's a `toLowerCase` on both strings

Reply:
Resolving this as I think it's a non-issue